meta-iotqa/lib/oeqa/runtime/peripherals/mraa/mraa_gpio.py

```diff
@@ -18,30 +34,75 @@ def setUp(self):
         else:
             raise unittest.SkipTest(output)
 
-    def test_gpio(self):
+    def check_gpio_value(self):
+        cmd = "cat /sys/class/gpio/gpio20/value".split()
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        return int(output)
+
+    def mraa_gpio_get_value(self):
+        status, output = self.target.run("mraa-gpio get " + self.pin)
+        output = str(output)
+        string1, string2 = output.split("=")
+        string2 = int(re.search(r'\d+', string2).group())
```

Comment:
This part reads the output of mraa-gpio get <pin number> on DUT 
which will be like PIN 25 = 1 
I am extracting value from string and returning it .
I am writing value from Host and reading it on DUT using above command .
I am adding status check for self.target.run("mraa-gpio get " + self.pin)